### DISCUSSION TITLE
Нужна помощь срочно. Как обнаружить прослушку и шпионов на телефоне?

### DISCUSSION THREAD

**Discussion by shaman06051978** [community/community#147760] [Accessibility]
### Select Topic Area

Question

### GitHub Feature Area

Discussions

### Body

Как обнаружить прослушку и шпионов на телефоне. И с помощью его как удалится из соц сетей? Помогите. Я профан. Есть бескорыстные еще хорошие люди кто поможет?

**Comment by shaman06051978** [comment]
Help

**Comment by shinybrightstar** [comment]
Hi @shaman06051978 , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
